src/env.h

```diff
@@ -973,9 +972,22 @@ struct EnvSerializeInfo {
 };
 
 struct SnapshotData {
-  v8::StartupData blob;
+  // The result of v8::SnapshotCreator::CreateBlob() during the snapshot
+  // building process.
+  v8::StartupData v8_snapshot_blob_data;
+
+  static const size_t kNodeBaseContextIndex = 0;
+  static const size_t kNodeMainContextIndex = kNodeBaseContextIndex + 1;
+
   std::vector<size_t> isolate_data_indices;
+  // TODO(joyeecheung): there should be a vector of env_info once we snapshot
+  // the worker environments.
   EnvSerializeInfo env_info;
+  // A vector of built-in ids and v8::ScriptCompiler::CachedData, this can be
+  // shared across Node.js instances because they are supposed to share the
+  // read only space. We use the vector because v8::ScriptCompiler::CachedData
```

Comment:
If `v8::ScriptCompiler::CachedData` is copyable we could've just used the list initializer with it, but it isn't so we use a custom type here.

Reply:
Yeah, then probably we can rephrase as:

```suggestion
  // read only space. We use native_module::CodeCacheInfo because v8::ScriptCompiler::CachedData
```